### PR TITLE
Update yarn commands with npm-run-all

### PR DESCRIPTION
**Pull request by kulla**
Have a more colorful output for `yarn format` and `yarn lint`. See for example:

2022-11-07-231718_931x301_scrot

### CODE REVIEW

**Comment by elbotho** [comment]
I appreciate the colors, but it creates a lot more irrelevant console output for me… so I think I prefer the current version. 

this pr:
```
[lint:eslint  ] 
[lint:eslint  ] > @serlo/serlo.org-cloudflare-worker@0.0.0 lint:eslint /Users/work/Documents/GitHub/cloudflare-worker/package.json
[lint:eslint  ] > yarn _eslint 
[lint:eslint  ] 
[lint:prettier] 
[lint:prettier] > @serlo/serlo.org-cloudflare-worker@0.0.0 lint:prettier /Users/work/Documents/GitHub/cloudflare-worker/package.json
[lint:prettier] > yarn _prettier --check 
[lint:prettier] 
[lint:tsc     ] 
[lint:tsc     ] > @serlo/serlo.org-cloudflare-worker@0.0.0 lint:tsc /Users/work/Documents/GitHub/cloudflare-worker/package.json
[lint:tsc     ] > tsc --noEmit 
[lint:tsc     ] 
[lint:prettier] Checking formatting...
[lint:prettier] All matched files use Prettier code style!
```

vs. main:
```
Checking formatting...
All matched files use Prettier code style!
```

**Comment by kulla** [comment]
> I appreciate the colors, but it creates a lot more irrelevant console output for me… so I think I prefer the current version.

Okay, I removed the arguments to `run-p` so the behavior stays the same.

File: package.json [hunk 1]
```diff
@@ -13,10 +13,10 @@
     "check:all": "scripts/run_all_checks.sh",
     "dev": "wrangler dev --env staging",
     "deploy": "wrangler publish",
-    "format": "npm-run-all -c \"format:*\"",
+    "format": "run-p -c \"format:*\"",
```
Comment (elbotho) [inline package.json:16]: Did you check if running eslint and prettier in parallel works without problems?
(run-p is parallel by default)

**Review by hugotiburtino — APPROVED** [review]
old and new ways are ok for me.

**Pull request merged**